GPU/GPUTracking/Merger/GPUTPCGMO2Output.cxx

```diff
@@ -134,12 +136,18 @@ GPUdii() void GPUTPCGMO2Output::Thread<GPUTPCGMO2Output::output>(int nBlocks, in
     if (merger.Param().par.dodEdx) {
       oTrack.setdEdx(tracksdEdx[i]);
     }
+
     oTrack.setOuterParam(o2::track::TrackParCov(
       outerPar.X, outerPar.alpha,
       {outerPar.P[0], outerPar.P[1], outerPar.P[2], outerPar.P[3], outerPar.P[4]},
       {outerPar.C[0], outerPar.C[1], outerPar.C[2], outerPar.C[3], outerPar.C[4], outerPar.C[5],
        outerPar.C[6], outerPar.C[7], outerPar.C[8], outerPar.C[9], outerPar.C[10], outerPar.C[11],
        outerPar.C[12], outerPar.C[13], outerPar.C[14]}));
+
+    const auto pid = pidResponse.getMostProbablePID(oTrack);
```

Comment:
This should not run unconditionally, since we don't necessarily need it in sync reco.
Does it require dEdx? If yes, please move it into the above if stratement `if (merger.Param().par.dodEdx) {`.
Otherwise, we should add an extra flag in the GPU reco settings to enable / disable.

Reply:
Yes, it needs the dEdx information. I added the if statement below otherwise, `mOuterParam.mPID` is overwritten by `oTrack.setOuterParam(o2::track::TrackParCov(`.